minecraft_servers/claymcnetwork/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "claymc",
+  "nice_name": "ClayMC",
+  "direct_ip": "mc.claymc.net",
+  "server_wildcards": [
+    "%.claymc.net",
+    "cactuscanyon.net",
+    "mc.clueserv.de",
+    "kuschelstube.net",
+    "qsgstube.de"
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://www.claymc.net",
+    "web_shop": "https://shop.claymc.net",
+    "twitter": "ClayMCNetwork",
+    "discord": "https://discord.claymc.net",
```

Comment:
Discord link has to be https//discord.gg/[INVITE]; custom domain redirect is reserved for partners